cmd/state/internal/cmdtree/builds.go

```diff
@@ -5,11 +5,12 @@ import (
 	"github.com/ActiveState/cli/internal/locale"
 	"github.com/ActiveState/cli/internal/primer"
 	"github.com/ActiveState/cli/internal/runners/builds"
+	"github.com/ActiveState/cli/pkg/project"
 )
 
 func newBuildsCommand(prime *primer.Values) *captain.Command {
 	runner := builds.New(prime)
-	params := &builds.Params{}
+	params := &builds.Params{Namespace: &project.Namespaced{}}
```

Comment:
```suggestion
	params := &builds.Params{}
```

This feels like an anti-pattern. Either it's nil or it's a real namespace, not a namespace struct with no values.

Reply:
Yep, that's exactly what I did first. However, if you supply a namespace, State Tool errors out due to https://github.com/ActiveState/cli/blob/cf6492853a1dfc5b10112b11ac192e3bdf273822/pkg/project/namespace.go#L45-L47

As a result, we need this. We do it for other commands too.

Reply:
Ahh I didn't realize we were doing this. Still something we should stop doing but fair enough; not in scope for this story.